src/external/libcurl.m4

```diff
@@ -2,3 +2,15 @@ AC_SUBST(CURL_LIBS)
 AC_SUBST(CURL_CFLAGS)
 
 PKG_CHECK_MODULES([CURL], [libcurl], [found_libcurl=yes], [found_libcurl=no])
+
+AC_MSG_CHECKING([whether libcurl knows CURLOPT_PROTOCOLS_STR])
+
+AC_LINK_IFELSE(
+    [AC_LANG_SOURCE([
+#include <curl/curl.h>
+int main () {
+    return CURLOPT_PROTOCOLS_STR;
```

Comment:
This works.

But just out of curiosity, wouldn't it be better to detect version using pkg-config?
```
$ pkg-config --modversion libcurl
8.0.1
```
and then ifdef based on lib version?

Reply:
Hi,

it picked the feature test to cover backports of this option to older versions as well. But I do not know if this is done by some distributions or not.

bye,
Sumit